_posts/2021-06-21-homebrew-3.2.0.md

```diff
@@ -0,0 +1,31 @@
+---
+title: 3.2.0
+author: MikeMcQuaid
+redirect_from: /blog/3.2.0/
+---
+Today I'd like to announce Homebrew 3.2.0. The most significant changes since 3.1.0 are `brew install` now upgrades outdated formulae by default and basic macOS 12 (Monterey) support.
+
+Major changes and deprecations since 3.1.0:
+
+- [`brew install` now upgrades outdated formulae by default (for indempotency). This can be disabled by setting [`HOMEBREW_NO_INSTALL_UPGRADE`].](https://github.com/Homebrew/brew/pull/11525)
```

Comment:
```suggestion
- [`brew install` now upgrades outdated formulae by default (for idempotency). This can be disabled by setting `HOMEBREW_NO_INSTALL_UPGRADE`.](https://github.com/Homebrew/brew/pull/11525)
```

Typo, and the extra square brackets looked weird.